cli/admin/README.md

```diff
@@ -122,9 +134,18 @@ Initial configuration:
 {"scopes":[{"scopeName":"_system"},{"scopeName":"streamsinscope"}]}
 ```
 
+The admin level pravega-cli needs to be accessed from a pod that has access to the bookkeeper and zookeeper such as the segmentstore pod.
```

Comment:
Here you are talking about "pods", but this part of the README is general and not specific to Kubernetes. If I remember correctly, the idea as to add a new subsection and explain "step-by-step" how to use the Pravega CLI on that environment. But let's do not mix Kubernetes-specific stuff in the general section of the CLI.